File: Lv.1/같은 숫자는 싫어.cpp
#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(vector<int> arr) 
{
    vector<int> answer;
    int cur=arr[0];
    answer.push_back(cur);
    for(int i=1;i<arr.size();i++)
    {
        if(cur==arr[i])
        {
            continue;
        }
        else
        {
            answer.push_back(arr[i]);
            cur=arr[i];
        }
        
        
    }
    // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    cout << "Hello Cpp" << endl;

    return answer;
}

File: Java/LV1/같은 숫자는 싫어.java
import java.util.*;

public class Solution {
    public int[] solution(int []arr) {
        int[] answer = {};
        List<Integer> temp = new ArrayList<>();
        temp.add(arr[0]);
        int cur = arr[0];
        for(int i=1;i<arr.length;i++){
            if(cur==arr[i]){
                continue;
            }
            else{
                temp.add(arr[i]);
                cur=arr[i];
            }
        }

        answer = new int[temp.size()];
        for(int i=0;i<temp.size();i++){
            answer[i]=temp.get(i);
        }
        return answer;
    }
}
